pkg/notifiers/sender_test.go

```diff
@@ -0,0 +1,29 @@
+package notifiers
+
+import (
+	"testing"
+
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsHTTPClientConfigSet(t *testing.T) {
```

Comment:
The tests should be setup with a list of cases to process, this helps make the tests easier to read and add additional tests later. An example can be seen here https://github.com/rancher/rancher/blob/master/pkg/rbac/common_test.go